manifest.json

```diff
@@ -1,11 +1,8 @@
 {
-    "manifest_version": 2,
+    "manifest_version": 3,
```

Comment:
chrome dev tools complained about manifest version 2 - that's why I also removed the background.js and renamed the action.

Reply:
Hi @raphaelbauer Thank you so much for this. I will look into this!